libs/community/langchain_community/tools/scenexplain/tool.py

```diff
@@ -21,8 +21,9 @@ class SceneXplainTool(BaseTool):
     name: str = "image_explainer"
     description: str = (
         "An Image Captioning Tool: Use this tool to generate a detailed caption "
-        "for an image. The input can be an image file of any format, and "
-        "the output will be a text description that covers every detail of the image."
+        "for an image. The input can be an image file of any format such as a URL or "
+        "a local file path, no other text, and the output will be a text description "
```

Comment:
a local file should not work and does not seem to work based on the api wrapper?